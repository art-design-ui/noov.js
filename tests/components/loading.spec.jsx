import React from 'react'
import { render } from '@testing-library/react'
import Loading from '../../src/client/components/loading'

describe('test Loading component', () => {
  it('should render the correct default loading', () => {
    const wrapper = render(<Loading />)
    const element = wrapper.container.querySelector('.loading')
    expect(element).toBeInTheDocument()
    expect(element.tagName).toEqual('DIV')
    expect(element.className).toEqual('loading')
    expect(element.innerHTML).toEqual('loading......')
  })
})
