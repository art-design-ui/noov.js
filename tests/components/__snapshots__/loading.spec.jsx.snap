// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Loading component should render the correct default loading 1`] = `
<div
  class="loading"
>
  loading......
</div>
`;
